autode/neb/original.py

```diff
@@ -244,32 +259,47 @@ def get_force(
         ) * hat_tau
 
         # ∇V(x)_i|_|_ = ∇V(x)_i - (∇V(x)_i•τ) τ
-        grad_perp = self.grad - np.dot(self.grad, hat_tau) * hat_tau
+        grad_perp = self.gradient - np.dot(self.gradient, hat_tau) * hat_tau
 
         # F_i = F_i^s|| -  ∇V(x)_i|_|_
         return f_parallel - grad_perp
 
+    @property
+    def gradient(self) -> Optional[np.ndarray]:
+        return None if self._grad is None else self._grad.flatten()
+
+    @gradient.setter
+    def gradient(self, value: Optional[np.ndarray]):
+        self._grad = None if value is None else value.flatten()
+
+    def _clear_energies_gradient_hessian(self) -> None:
+        """Skip any clearing energies"""
+
 
 class Images(Path):
-    def __init__(self, num, init_k, min_k=None, max_k=None):
+    def __init__(
+        self,
+        init_k: ForceConstant,
+        min_k: Optional[ForceConstant] = None,
+        max_k: Optional[ForceConstant] = None,
+    ):
         """
         Set of images joined by harmonic springs with force constant k
 
         -----------------------------------------------------------------------
         Arguments:
 
-            num (int): Number of images
-
-            init_k (float): Initial force constant (Ha Å^-2)
+            init_k (ForceConstant): Initial force constant
 
-            min_k (None | float): Minimum value of k
+            min_k (ForceConstant | None): Minimum value of k
 
-            max_k (None | float): Maximum value of k
+            max_k (ForceConstant | None): Maximum value of k
         """
-        super().__init__(*(Image(name=str(i), k=init_k) for i in range(num)))
+        super().__init__()
 
-        self.min_k = init_k / 10 if min_k is None else float(min_k)
-        self.max_k = 2 * init_k if max_k is None else float(max_k)
+        self.init_k = init_k
+        self.min_k = init_k / 10 if min_k is None else min_k
+        self.max_k = 2 * init_k if max_k is None else max_k
```

Comment:
Is it necessary to assert max_k > min_k ?

Reply:
absolutely – thanks 😄